owid/datautils/dataframes.py

```diff
@@ -1,17 +1,45 @@
 """Objects related to pandas dataframes."""
 
-import inspect
 import warnings
-from pathlib import Path
 from typing import Any, Callable, Dict, List, Optional, Tuple, Union, cast
 
 import numpy as np
 import pandas as pd
 from pandas.api.types import union_categoricals
+from owid.datautils.io.df import to_file as to_file_, has_index as has_index_
 
 from owid.datautils.common import ExceptionFromDocstring, warn_on_list_of_entities
 
 
+# Backwards compatibility
+def to_file(*args: Any, **kwargs: Any) -> None:
+    """Save a dataframe in any format.
+
+    Will be deprecated. Use owid.datautils.io.df.to_file instead.
+    """
+    warnings.warn(
+        "Call to deprecated class to_file (This function will be removed in the next"
+        " minor update, use owid.datautils.io.df_to_file instead.)",
+        category=DeprecationWarning,
+        stacklevel=2,
+    )
+    to_file_(*args, **kwargs)
+
+
+def has_index(*args: Any, **kwargs: Any) -> bool:
+    """Save a dataframe in any format.
```

Comment:
This docstring is wrong. I think `has_index` makes much more sense as part of `dataframes`, doesn't it? I suppose you moved it to avoid importing dataframes from io.df? No big deal anyway.

Reply:
It seemed to me that `has_index` was an auxiliary function for `to_file`, as no other method depended on it. 

But I agree that if this is a function that we want to expose to the end user, then it should live in `dataframes`.